app/src/components/Register.tsx

```diff
@@ -42,20 +46,25 @@ class Register extends React.Component<RegisterProps, RegisterState> {
       artistBirthYear: '',
       createdDate: '',
       medium: '',
+      edition: '',
+      artworkCreationDate: '',
+      imageIpfsHash: '',
       size: '',
     };
 
     this.registerArtifact = this.registerArtifact.bind(this);
     this.getRegisterTransactionStatus = this.getRegisterTransactionStatus.bind(this);
+
+    this.captureFile = this.captureFile.bind(this);
```

Comment:
Change capture file from a function definition to a property e.g. captureFile = (params) => {}, as this autobinds which makes the constructor much nicer.